test/integration/manifests/goldpinger/cluster-role-binding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: goldpinger-serviceaccount
-    namespace: default
+    namespace: linux-datapath-test
```

Comment:
Same with this, I think the same yaml is used for different test. So let's try to keep them separate in case we are running both on the same cluster.